client/extensions/wp-job-manager/components/settings/pages/page-dropdown.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ReduxFormSelect from 'components/redux-forms/redux-form-select';
+
+const PageDropdown = ( { name, pages, ...otherProps } ) => {
+	return (
+		<ReduxFormSelect name={ name } { ...otherProps }>
+			{ pages.map( ( { ID, title } ) => (
+				<option key={ ID } value={ ID }>
+					{ title }
+				</option>
+			) ) }
+		</ReduxFormSelect>
+	);
+};
```

Comment:
I thought about doing that because it makes sense to define the prop where it's used, but that would mean that `getSitePosts` would execute 3x (once per dropdown) instead of once in the parent. Do you think the former benefit outweighs the latter?